app/util/conf.py

```diff
@@ -99,6 +99,7 @@ class BambooSettings(BaseAppSettings):
     def __init__(self, config_yml):
         super().__init__(config_yml)
         self.concurrency = self.get_property('concurrency')
+        self.webdriver_visible = self.get_property('WEBDRIVER_VISIBLE')
         self.total_actions_per_hour = self.get_property('total_actions_per_hour')
         self.ramp_up = self.get_property('ramp-up')
```

Comment:
pls doulbe check if `ramp_up` (and all other settings) are used in bamboo scripts. Otherwise just remove it, no need to introduce anything which is not used yet.